src/singletons/draw-manager/earth.ts

```diff
@@ -626,7 +626,7 @@ export class Earth {
     const float latitudeCenter = 67.5; // The latitude at which the Aurora Borealis appears
     const float latitudeMargin = 7.0; // The margin around the center latitude where the Aurora Borealis is visible
     const vec3 directionalLightColor = vec3(1.0, 1.0, 1.0);
-    const vec3 ambientLightColor = vec3(0.1, 0.1, 0.1);
+    const vec3 ambientLightColor = vec3(0.15, 0.15, 0.15);
```

Comment:
OPTIONAL: This will increase the overall brightness of the earth (you can reduce it to make everything darker)

Reply:
I wanted to point it out since you noted that Spain appeared darker than expected.